provisionersdk/proto/provisioner.proto

```diff
@@ -132,7 +132,7 @@ message Resource {
     bool hide = 5;
     string icon = 6;
     string instance_type = 7;
-    int32 cost = 8;
+    int32 daily_cost = 8;
```

Comment:
I'm sure there was a conversation here, but why we did land on "daily" as the unit? I've been thinking about how we present this, and the more I thought about it the more I think giving this a unit is not ideal. Mainly because of it being lossy to our currently planned implementation.

Reply:
We want to encourage standard practices across all deployments for follow-on features such as reporting and burst quotas. For time units, the reasonable options are "hourly", "daily" and "monthly".

I threw out hourly because I thought it may require precision beyond a cent. For example, a $20/m instance costs 2.7 cents per hour, and we want to avoid floating point arithmetic.

I threw out monthly because it's not always clear what the exact length of a month is. It could be 30, 30.5, or 30.437 (right answer) days.